Monika After Story/game/script-farewells.rpy

```diff
@@ -1460,27 +1460,110 @@ init 5 python:
     )
 
 label bye_prompt_housework:
-    if mas_isMoniNormal(higher=True):
-        m 1eub "Doing your chores, [player]?"
-        m 1ekc "I would like to help you out, but there's not really much I can do since I'm stuck in here..."
-        m 3eka "Just make sure to come back as soon as you're done, okay?"
-        m 3hub "I'll be waiting here for you~"
-    elif mas_isMoniUpset():
-        m 2esc "Fine."
-        m 2tsc "At least you're doing something responsible."
-        m 2tfc "{cps=*2}...For once.{/cps}{nw}"
+#TODOS: Chance for upset/distressed to not want to go with you, if not I'd ideally like it to default to regular goodbye
+# How to set it to non-generic "we're home" greeting?
+    if persistent.mas_carryme_choice = mas.dockstat.CM_TAKE_MONI
+        if mas_isMoniNormal(higher=True):
+            m "You've got some things to get done around the house?"
+            m "I'm glad you're taking good care of your space, [player]."
+            m "Hang on, I'll come with you!"
+            jump mas_dockstat_iostart
+        elif mas_isMoniUpset():
+            m "Oh, you're doing some work?"
+            m "...Well, I'm glad you're being productive, at least."
+            m "I'm going to make sure you're not slacking off, though."
+            jump mas_dockstat_iostart
+        elif mas_isMoniDis():
+            m "...I want to come with."
+            m "I need to get out of this room, I don't really care if we're not going anywhere."
+            jump mas_dockstat_iostart
+
+    elif persistent.mas_carryme_choice = mas.dockstat.CM_ASK_MONI
+        if mas_isMoniNormal(higher=True):
+        m "Oh, you've got some chores to get done?"
+        m "I know there's not much I can do to help out, but..."
+        m "Could I come along?{nw}"
         $ _history_list.pop()
-        m 2esc "Goodbye."
-    elif mas_isMoniDis():
-        m 6ekc "I see..."
-        m 6rkc "I don't want to keep you from completing your household responsibilities."
-        m 6dkd "I just hope you're actually busy and not saying that just to get away from me..."
-        m 6ekc "Goodbye, [player]."
-    else:
-        m 6ckc "..."
-    $ persistent._mas_greeting_type = store.mas_greetings.TYPE_CHORES
-    $ persistent._mas_greeting_type_timeout = datetime.timedelta(hours=5)
-    return 'quit'
+        menu:
+            m "Could I come along?{fast}"
+
+            "Yes."
+                m "Okay!"
+                m "I'll do my best to provide moral support!"
+                m "Give me just a second to get ready..."
+                jump mas_dockstat_iostart
+
+            "No."
```

Comment:
Unsure if we want to have an "I can't take you with me right now" option for the sake of having an additional 'rude' option (loses affection ofc), might be excessive though.

```suggestion
            "No.":
```